addons/shortcodes.php

```diff
@@ -203,16 +221,52 @@ function shortcode_attendees( $attr ) {
 			</ul>
 		</div>
 		<br class="tix-clear" />
+
 		<?php
 		$this->log( sprintf( __( 'Generated attendees list in %s seconds', 'camptix' ), microtime(true) - $start ) );
 		wp_reset_postdata();
+
 		$content = ob_get_contents();
 		ob_end_clean();
+
 		set_transient( $transient_key, array( 'content' => $content, 'time' => time() ), $cache_time );
+
 		return $content;
 	}
 
 	/**
+	 * Generate an avatar placeholder element with a data attribute that contains
+	 * the Gravatar hash so the real avatar can be loaded asynchronously.
+	 *
+	 * @param $id_or_email
+	 *
+	 * @return string
+	 */
+	protected function get_avatar_placeholder( $id_or_email ) {
+		// @todo Allow customization of avatar and placeholder size
+		$size = 96;
+
+		return sprintf(
+			'<div class="avatar avatar-placeholder" data-url="%s" data-url2x="%s" data-size="%s" data-alt="%s" data-appear-top-offset="500"></div>',
```

Comment:
This line, and the `<img>` tag below, would be more readable if the attributes were broken up onto multiple lines.